extensions/integration-tests/src/notebook.test.ts

```diff
@@ -111,4 +99,16 @@ async function openNotebook(content: azdata.nb.INotebookContents, kernelMetadata
 	assert(notebook !== undefined && notebook !== null, 'Expected notebook object is defined');
 	return notebook;
 }
+async function verifyClearAllOutputs(notebook: azdata.nb.NotebookEditor) {
+	let cellWithOutputs = notebook.document.cells.find(cell => cell.contents && cell.contents.outputs && cell.contents.outputs.length > 0);
+	assert(cellWithOutputs !== undefined, 'Could not find notebook cells with outputs');
+	console.log('Before clearing cell outputs');
+	let clearedOutputs = await notebook.clearAllOutputs();
+	let cells = notebook.document.cells;
+	cells.forEach(cell => {
+		assert(cell.contents && cell.contents.outputs && cell.contents.outputs.length === 0, `Expected Output: 0, Actual: '${cell.contents.outputs.length}'`);
```

Comment:
Would be useful to print out the actual contents of the cell instead of just the length here for debugging